Formula/soapysdr.rb

```diff
@@ -0,0 +1,47 @@
+class Soapysdr < Formula
+  desc "Vendor and platform neutral SDR support library."
+  homepage "https://github.com/pothosware/SoapySDR/wiki"
+  url "https://github.com/pothosware/SoapySDR/archive/soapy-sdr-0.8.0.tar.gz"
+  sha256 "cff3b1fb1cd3b88932f37df42f69fc3d8e0ea42f4788b35131ceaf5d1b908ac2"
+  license "BSL-1.0"
+  revision 1
+  head "https://github.com/pothosware/SoapySDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.9"
+
+  def install
+
+    args = []
+
+    args += ["-DENABLE_PYTHON=OFF"]
+    args += ["-DENABLE_PYTHON3=ON"]
+    args += ["-DCMAKE_MODULE_LINKER_FLAGS_INIT='-undefined dynamic_lookup'"]
+
+    if !(build.head?)
+      args += ["-DSOAPY_SDR_EXTVER=release"]
+    end
+
+    args += %W[-DSOAPY_SDR_ROOT='#{HOMEBREW_PREFIX}']
+
+    mkdir "build" do
+      args += std_cmake_args
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    mkdir build
+    cd build
+    cmake ..
+    cmake --build . --config %CONFIGURATION%
+    cmake --build . --config %CONFIGURATION% --target install
+
+    SoapySDRUtil --info
+    SoapySDRUtil --check=null
+    SoapySDRUtil --make="driver=null"
+    ctest
```

Comment:
This isn't valid Ruby code. The test you've written reads like a shell script, when this needs to be written in Ruby. The test block is run in an empty temporary directory, so the calls to `cmake` will just error out.

Reply:
@99Edord